leetcode/1008.cpp

```diff
@@ -56,3 +58,9 @@ class Solution {
         return root;
     }
 };
+
+int main(){
+  Solution solution;
+  Solution2 solution2;
+  return 0;
```

Comment:
Maybe you can complete the input and output and how to call the solve function in the main function to make this program runnable.